python-with-pandas/3.6-alpine/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.6-alpine3.9
+
+LABEL maintainer=pcrespov
+
+RUN apk add --no-cache --virtual .BUILD-DEPS \
```

Comment:
That is very cool!

But I read somewhere and just tested that:
having the deletion in a separate RUN will not reduce the overall image size because it is in a separate layer.
it basically cleans what is in the final image, but the image size remains the same...

Reply:
I see. I did not know that. 
I also wonder if removing these libraries at the last step invalidates caching of the installation (i.e. libraries are not there anymore) ... 

I guess we need to check these things

Reply:
Have a look at this [Lighter Python images using multi-stage Dockerfiles](https://lekum.org/post/multistage-dockerfile/)

Reply:
done